CMakeLists.txt

```diff
@@ -78,6 +78,7 @@ option(IREE_BUILD_SAMPLES "Builds IREE sample projects." ON)
 option(IREE_BUILD_PYTHON_BINDINGS "Builds the IREE python bindings" OFF)
 option(IREE_BUILD_TRACY "Enables building the 'iree-tracy-capture' CLI tool and includes it in runtime Python bindings." OFF)
 option(IREE_BUILD_BUNDLED_LLVM "Builds the bundled llvm-project (vs using installed)" ON)
+option(IREE_USE_SYSTEM_DEPS "Uses system libraries instead of building bundled or downloaded dependencies" OFF)
```

Comment:
I think needs some additional words, including a warning  that this untested and that it is up to the user to ensure that compatible versions are available.

Reply:
I am perfectly fine with what @ScottTodd suggested :)